FreeRTOS/FreeRTOS
# Update all CMock configuration to compare_ptr.

Description
-----------

Follow [PR#1121](https://github.com/FreeRTOS/FreeRTOS/pull/1121), bring the change of CMock configuration in directory tasks to all other directories.

Test Steps
-----------

`cd FreeRTOS/Test/CMock`
`make lcov`

Checklist:
----------



Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
